### PR TITLE
Add documentation and example code

### PR DESCRIPTION
**Pull request by melmccann**

### CODE REVIEW

File: src/SmartboxSkeletonRemoteDemoBundle/Controller/DefaultController.php [hunk 1]
```diff
@@ -24,4 +25,21 @@ public function pongAction()
...
+     * @return Response
+     */
+    public function logContentAction(Request $request)
+    {
+        $logger = $this->get('logger');
+        $content = $request->getContent();
+        $logContent = \json_encode($content);
+        $logger->info($logContent);
+
+        return new Response('{"status":"ok"}', Response::HTTP_OK, array('Content-Type' => 'application/json'));
```
Comment (GaryPEGEOT) [inline src/SmartboxSkeletonRemoteDemoBundle/Controller/DefaultController.php]: new JsonResponse(['status' => 'ok']);

Comment (melmccann) [reply src/SmartboxSkeletonRemoteDemoBundle/Controller/DefaultController.php]: Thanks @GaryPEGEOT 
I will update this.

File: src/SmartboxSkeletonBundle/Controller/ApiController.php [hunk 2]
```diff
@@ -56,12 +62,36 @@ public function apiAction(Request $request, $methodName)
...
+     * @return Response
+     */
+    public function pingCallbackAction(Request $request)
+    {
+        $logger = $this->get('logger');
+        $content = $request->getContent();
+        $logContent = \json_encode($content);
+        $logger->info($logContent);
+
+        return new Response('{"status":"ok"}', Response::HTTP_OK, ['Content-Type' => 'application/json']);
```
Comment (GaryPEGEOT) [inline src/SmartboxSkeletonBundle/Controller/ApiController.php]: JsonResponse here too

File: src/SmartboxSkeletonRemoteDemoBundle/Controller/DefaultController.php [hunk 3]
```diff
@@ -5,7 +5,12 @@
 namespace SmartboxSkeletonRemoteDemoBundle\Controller;
 
 use SmartboxSkeletonBundle\Entity\PingMessage;
+<<<<<<< HEAD
```
Comment (GaryPEGEOT) [inline src/SmartboxSkeletonRemoteDemoBundle/Controller/DefaultController.php]: Oh Mel

File: src/SmartboxSkeletonRemoteDemoBundle/Controller/DefaultController.php [hunk 4]
```diff
@@ -26,4 +31,19 @@ public function pongAction()
 
         return new Response($json, 200, ['Content-Type' => 'application/json']);
```
Comment (GaryPEGEOT) [inline src/SmartboxSkeletonRemoteDemoBundle/Controller/DefaultController.php]: JsonResponse everywhere

File: src/SmartboxSkeletonRemoteDemoBundle/Controller/DefaultController.php (hunk 3 above)
Comment (melmccann) [reply src/SmartboxSkeletonRemoteDemoBundle/Controller/DefaultController.php]: ha ha ha.. how did that slip though!!! I will fix it now :-D

File: src/SmartboxSkeletonRemoteDemoBundle/Controller/DefaultController.php (hunk 4 above)
Comment (melmccann) [reply src/SmartboxSkeletonRemoteDemoBundle/Controller/DefaultController.php]: I am going to leave that the way it is for now as I can't seem to get JMS to normalise the entity to an array for the response. We can revisit this at another time :-D

**Pull request merged**